mysql-test/suite/rpl/t/rpl_row_triggers_sbr.test

```diff
@@ -0,0 +1,37 @@
+--source include/have_binlog_format_statement.inc
+--source include/have_rbr_triggers.inc
+--source include/master-slave.inc
+
+--disable_query_log
+CALL mtr.add_suppression("Cannot execute statement: impossible to write to binary log since statement is in row format and BINLOG_FORMAT = STATEMENT");
+--enable_query_log
+
+set binlog_format = row;
+create table t1 (i int);
+create table t2 (i int);
+
+--sync_slave_with_master
+
+--disable_query_log
+CALL mtr.add_suppression("impossible to write to binary log since statement is in row format and BINLOG_FORMAT = STATEMENT");
```

Comment:
Didn't add_suppression on master replicate to slave too? (the texts are different too, the shorter one can be used)